deploy/monitor/prometheus_rule.yaml

```diff
@@ -23,13 +23,13 @@ spec:
           description: "The UnifiedPush Operator has been down for more than 5 minutes. "
           summary: "The UnifiedPush Operator is down. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
           sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc"
-      - alert: UnifiedpushPodcount
+      - alert: UnifiedPushPodcount
         annotations:
-          description: "The Pod count for the UnifiedPush has changed in the last 5 minutes."
-          summary: "Pod count for namespace UnifiedPush is {{ printf '%.0f' $value }}. Expected 3 pods. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
+          description: "The Pod count for the UnifiedPush has changed in the last 5 minutes and is lower than the minimal quantity required."
+          summary: "Pod count for namespace UnifiedPush is {{ printf '%.0f' $value }}. Expected 3 pods at least. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
           sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc"
         expr: |
-          (1-absent(kube_pod_status_ready{condition="true", namespace="unifiedpush"})) or sum(kube_pod_status_ready{condition="true", namespace="unifiedpush"}) != 3
+          (1-absent(kube_pod_status_ready{condition="true", namespace="unifiedpush"})) or sum(kube_pod_status_ready{condition="true", namespace="unifiedpush"}) < 3
```

Comment:
Again, there's a hard-coded namespace value here. At least in integreatly/RHMI, it's not going to be called "unifiedpush", and we shouldn't presume to know the name of it upfront like this. Is it possible to template this here (and any other instances of it)?

Reply:
@grdryn tks for your input but I think that we already spoke over it in another PR. 
Note that it has been doing as it is done in other projects which were the agreement. Also, in RHMI it will be installed as a template and changed there by ansible scripts. I'd recommend you check it with @austincunningham and @laurafitzgerald which worked previously in these tasks.

Reply:
If you're sure, then that's fine. Could you link to the previous conversation that you're talking about, or something that shows where it's templated?

Reply:
Hi @grdryn,

As I said before I am just doing as it was done before by them. So, it should work in the same way.  However, off the curse, you are able to discuss it and do improvements as please you in the future, which indeed I believe that will be easier after has all dashboards and rules defined and working well which is our goal here. You can check one example over how these integrations are made and implemented in integr8ly here https://github.com/integr8ly/installation/pull/687/files#diff-eaa212d002e12d63341d8140d3803235R18.